pyproject.toml

```diff
@@ -18,27 +18,27 @@ packages = []
 [tool.poetry.dependencies]
 python = "^3.10.0"
 
-django = "==5.0.2"
-django-celery-beat = "==2.6.0"
-django-compressor = "==4.4"
-django-cors-headers = "==4.3.1"
+django = "==5.1.1"
```

Comment:
You've updated this file, but did not push `poetry.lock` file. You can run `poetry lock --no-update` to update the file

Reply:
This is what I was missing. Thanks.